backend/tests/test_email_client_smtp.py

```
@@ -1,14 +1,371 @@
+    monkeypatch.setattr(
+        "services.email_client._starttls_existing_transport",
+        fake_starttls_existing_transport,
+    )
+
+    result = await email_client._send_pinned_smtp_message(
+        message,
+        smtp_socket=smtp_socket,
+        smtp_server="smtp.example.com",
+        smtp_port=587,
+        smtp_username="testuser",
+        smtp_password="secret",
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to argument: "smtp_password"

(S106)